packages/smooth_app/lib/cards/data_cards/nutrition_level_card.dart

```diff
@@ -1,101 +1,35 @@
 import 'package:flutter/material.dart';
-import 'package:openfoodfacts/model/NutrientLevels.dart';
+import 'package:openfoodfacts/model/Attribute.dart';
+import 'package:smooth_app/cards/data_cards/nutrition_level_chip.dart';
 
 class NutritionLevelCard extends StatelessWidget {
-  const NutritionLevelCard(
-      {@required this.title,
-      this.subtitle,
-      @required this.level,
-      this.icon,
-      this.chip = false});
+  const NutritionLevelCard(this.attribute, this.iconWidth);
```

Comment:
I'm wondering about the name NutritionLevelCard, this widget is for any type of attribute group, right? If it is, should we rename it to AttributeGroupCard ?

Reply:
@stephanegigandet I just refactored a class that was already there; I agree that the name could be changed.
That being said, it's not exactly about `AttributeGroup` (this was not the point of the issue), it's about a list of `Attribute`s.

Therefore:
* as is, `NutritionLevelCard` could be renamed something like `AttributeCard` (which already exists, but the current `AttributeCard` is more or less a `NutritionLevelChip` - which could be rename `AttributeChip`) (in short, renaming is possible)
* for the moment we don't handle `AttributeGroup`s for this issue, and I don't think it would make sense for all end-users to use the order of our `init_attribute_groups_en.json`
* but we currently use hard-coded values for the display of `Attribute`s on the product page, which may not be the target

Reply:
A list of attributes is fine indeed.